daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/ledgerinteraction/SubmitError.scala

```diff
@@ -188,16 +193,6 @@ object SubmitError {
       )
   }
 
-  final case class ContractNotActive(templateId: Identifier, contractId: ContractId)
-      extends SubmitError {
-    override def toDamlSubmitError(env: Env): SValue =
-      SubmitErrorConverters(env).damlScriptError(
-        "ContractNotActive",
-        3,
-        ("contractId", fromAnyContractId(env.scriptIds, toApiIdentifier(templateId), contractId)),
-      )
-  }
-
```

Comment:
If you've removed the builder, we should also remove the variant from the daml code and update the indices. Remember we have no backwards compat yet - get it gone!

Reply:
done :)

Reply:
Gotta remove it from the show instance too :)